### ISSUE TITLE
Suddenly no more checkbox there

### ISSUE THREAD

**Issue by crazycat89** [stcr/subscribe-to-comments-reloaded#722]
II realized today that the checkbox suddenly disappeared completely. Is the issue known? Is it maybe related to the last update?
Link to my site: https://bit.ly/3Q24B2V
Thanks for any help

**Comment by mebishalnapit** [comment]
This plugin(StCR) adds the subscription functionality to the native WordPress commenting system only. And it does not work for 3rd party plugins that replace the commenting system with their own. And in your case, it's a different plugin which modifies the WordPress commenting system, so, it will not work for your site as you intended.

**Comment by crazycat89** [comment]
Yes, of course. I know … After it stops working, I replaced it by another Plugin.

**Comment by sympatrick** [comment]
> Okay. If you wanna give it another try then, please visit `StCR -> Options` section and just try once by clicking on the `Save Changes` button. And if none of the posts types are selected for `Enable on post types` option then, enable them and save the changes and see if that helps.

This resolved the problem for me.
